core/templates/dev/head/components/private_info_tooltip/private_info_tooltip_directive.html

```diff
@@ -0,0 +1,37 @@
+<div class="conversation-skin-tutor-card-tooltip">
+      <button class="conversation-skin-tutor-card-tooltip-button">?
```

Comment:
Should only indent by 2 from the previous line. (The next couple of lines need fixing, too.)

Reply:
Done, thanks!